### ISSUE TITLE
Use structured Qt abstractions for dealing paths

### ISSUE THREAD

**Issue by maxhaton** [ChipsandCheese/CnC_GUI#15]
Currently paths are built using strings, this is hard to refactor (i.e. the current behaviour is wrong)

**Comment by maxhaton** [comment]
@Nintonito

**Comment by maxhaton** [comment]
https://doc.qt.io/qt-5/qdir.html

**Comment by Nintonito** [comment]
Yeah this is a good issue to make.  I do want to change that over.  I will take a look.

**Comment by Nintonito** [comment]
See https://github.com/ChipsandCheese/CnC_GUI/commit/4fe7a420878ac3b3393b1491946af9d1d6c06e0a

I believe this is a decent improvement.  I will leave this issue open to solicit feedback.
